src/services/formatting/formattingScanner.ts

```diff
@@ -265,8 +267,7 @@ namespace ts.formatting {
 
         function isOnToken(): boolean {

```

Comment:
just for my own understanding, what is `isOnToken` supposed to do and also why were we checking if `startPos < endPos` before and are not checking it now?
I guess a pre-requisite question for that: what are `startPos` and `endPos` and the several `pos`s supposed to be?

Reply:
you answered the first part already in another comment, didn't see the comment when I was reviewing